modules/executable/01_mod.mk

```diff
@@ -12,8 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-exe_operating_systems ?= darwin,linux,windows
-exe_architectures ?= amd64,arm,arm64,ppc64le,s390x
+exe_targets ?= darwin_amd64_v1,darwin_arm64,linux_amd64_v1,linux_arm_6,linux_arm64,linux_ppc64le,linux_s390x,windows_amd64_v1,windows_arm_6,windows_arm64
```

Comment:
suggestion: we previously built for armv7, not armv6 as we have here: https://github.com/cert-manager/cert-manager/blob/876e386ee905aa86e2466c287e654613b0426927/make/cmctl.mk#L41

I'd vote to keep things the same as we had before in terms of which version we target

Reply:
updated